[2022-02-03 20:53:33,973] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2022-02-03 20:53:34,221] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2022-02-03 20:53:34,229] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:53:34,245] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-03 20:53:34,248] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:53:34,577] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-09-01 00:00:00+00:00
[2022-02-03 20:53:34,627] {standard_task_runner.py:52} INFO - Started process 6850 to run task
[2022-02-03 20:53:34,726] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_5', 'download_dataset_task', 'scheduled__2019-09-01T00:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_2.py', '--cfg-path', '/tmp/tmp_onxxp_j', '--error-file', '/tmp/tmp16ef0nsj']
[2022-02-03 20:53:34,734] {standard_task_runner.py:77} INFO - Job 43: Subtask download_dataset_task
[2022-02-03 20:53:35,568] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2019-09-01T00:00:00+00:00 [running]> on host 985445ce9491
[2022-02-03 20:53:36,703] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_5
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-09-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-09-01T00:00:00+00:00
[2022-02-03 20:53:36,721] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-03 20:53:36,728] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'pwd']
[2022-02-03 20:53:36,829] {subprocess.py:85} INFO - Output:
[2022-02-03 20:53:36,974] {subprocess.py:89} INFO - /tmp/***tmp0fqu8ibv
[2022-02-03 20:53:36,986] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-03 20:53:39,070] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_5, task_id=download_dataset_task, execution_date=20190901T000000, start_date=20220203T205334, end_date=20220203T205339
[2022-02-03 20:53:39,664] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 20:53:40,730] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
